js/mexc.js

```diff
@@ -1801,9 +1801,9 @@ module.exports = class mexc extends Exchange {
             // 'stopLossPrice': this.priceToPrecision (symbol, stopLossPrice),
             // 'takeProfitPrice': this.priceToPrecision (symbol, takeProfitPrice),
         };
-        const stopPrice = this.safeNumber2 (params, 'stopLossPrice', 'stopPrice');
+        const stopPrice = this.safeNumber (params, 'stopLossPrice');
```

Comment:
`stopPrice` is the unified parameter

Reply:
@samgermain `stopPrice` is a unified parameter for a different purpose: https://github.com/ccxt/ccxt/pull/12302#issuecomment-1065972543

Reply:
> @samgermain `stopPrice` is a unified parameter for a different purpose: [#12302 (comment)](https://github.com/ccxt/ccxt/pull/12302#issuecomment-1065972543)

I don't understand the difference between `stopPrice` and `stopLossPrice`

Reply:
`stopPrice` is specifically for stop-orders (which can be entry order, exit order or sometimes stoploss order too). so, stopLossPrice is specifically for those embeded child-order, which is tied to entry-order (when firing `createOrder`)